.tekton/run-acceptance-test.yaml

```diff
@@ -0,0 +1,85 @@
+    pipelinesascode.tekton.dev/task-5: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/generate-build-artifacts.yaml"
+    pipelinesascode.tekton.dev/task-6: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/generate-ids.yaml"
+    pipelinesascode.tekton.dev/task-7: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/get-cluster-version.yaml"
+    pipelinesascode.tekton.dev/task-8: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/get-component-versions.yaml"
+    pipelinesascode.tekton.dev/task-9: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/install-cluster.yaml"
+    pipelinesascode.tekton.dev/task-10: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/install-serverless-operator.yaml"
+    pipelinesascode.tekton.dev/task-11: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/mirror-and-tag-imagestreams.yaml"
+    pipelinesascode.tekton.dev/task-12: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/mirror-operator-images.yaml"
+    pipelinesascode.tekton.dev/task-13: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/release-tests.yaml"
+    pipelinesascode.tekton.dev/task-14: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/run-tfa-classifire.yaml"
+    pipelinesascode.tekton.dev/task-15: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/send-results-to-dashboard.yaml"
+    pipelinesascode.tekton.dev/task-16: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/send-slack-notification.yaml"
+    pipelinesascode.tekton.dev/task-17: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/setup-testing-accounts.yaml"
+    pipelinesascode.tekton.dev/task-18: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/uninstall-cluster.yaml"
+    pipelinesascode.tekton.dev/task-19: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/upload-artifacts.yaml"
+    pipelinesascode.tekton.dev/task-20: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/upstream-triggers-tests.yaml"
+    pipelinesascode.tekton.dev/task-21: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/upstream-pipelines-tests.yaml"
+    pipelinesascode.tekton.dev/task-22: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/upstream-pac-tests.yaml"
+    pipelinesascode.tekton.dev/task-23: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/upstream-operator-tests.yaml"
+    pipelinesascode.tekton.dev/task-24: "https://gitlab.cee.redhat.com/tekton/plumbing/-/raw/master/ci/tasks/whitelist-domains.yaml"
+    pipelinesascode.tekton.dev/max-keep-runs: "2"  
+spec:
+  pipelineRef:
+    name: acceptance-tests
+  params:
+    - name: GIT_RELEASE_TESTS_URI
```

Comment:
1. specify `CATALOG_SOURCE=custom-operators` because now it installs 1.15.1 instead of 5.0
2. reorder params alphabetically

Reply:
Done